input/postgres/relation_stats_aux.go

```diff
@@ -21,37 +21,43 @@ SELECT logicalrelid::oid,
  WHERE ($1 = '' OR (n.nspname || '.' || c.relname) !~* $1)
 `
 
+func collectCitusRelationStats(postgresVersion state.PostgresVersion, server *state.Server) bool {
+	return postgresVersion.IsCitus && server.Config.DisableCitusSchemaStats != "all"
+}
```

Comment:
This file best reviewed with whitespace diffs disabled. I decided to break the guards out into separate functions and use early returns to simplify the original function bodies.